loomi/regulariza-morador-client
# feature/sync-2

## Fix upgrade of indexeddb version 1 to version 2

---

**What I did:**

- Fix upgrade function of indexeddb version 1 to version 2
- Fix resume page problem with sync
- Add launch.json file to .vscode folder to help debug
- Add validations and helpers to deal with the logic stored in browers with indexeddb version 1

**How to test:**

1. Copy the branch test/dexie-version-1 and feature/sync-2
2. Copy .env, install dependencies and run `yarn start` the test branch
3. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
4. Register and play with the research, initiate, complete, update answers, go back to middle of research, make loops, do whatever you want
5. Stop the app and run the feature branch
6. Check if the answers and research in the stage are correct
7. Test the things that you think are worth testing, even the ones that are not in this description
